extensions/github/src/components/DiscussionListItem.tsx

```diff
@@ -0,0 +1,68 @@
+import { List, Icon, Color, ActionPanel, Action } from "@raycast/api";
+import { format } from "date-fns";
+
+import { DiscussionFieldsFragment } from "../generated/graphql";
+import { getGitHubUser } from "../helpers/users";
+
+function getDiscussionIcon(discussion: DiscussionFieldsFragment): List.Item.Props["icon"] {
+  const categoryText = discussion.category?.name ? `Category: ${discussion.category?.name}` : "Unknown";
+  const emojiHTML = discussion.category?.emojiHTML as string | null | undefined;
+  if (!emojiHTML) {
+    return { value: { source: discussion.repository?.owner?.avatarUrl }, tooltip: categoryText };
+  }
+  const fallbackImage = () => {
+    // note: getting the emoji directly break raycast. Looks like encoding issues.
```

Comment:
what does that mean?

Reply:
I will create a dedicated issue for that because this behavior is not only in this extension.

Reply:
right, let's try to fix it because it's a bit weird now (the code for the GH extension should aim to be an example for all extensions so we should avoid adding workarounds 😄 )